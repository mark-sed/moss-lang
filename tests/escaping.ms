a = true

if (a \
    == true) {
    "hello\n"
}

c = 43

fun \
foo(x,
y) {

    if (c == 42 \
        || c == 41 \
        || x == 5) {
        "YES\n"
    }
}

~foo(1, 5)
~foo(5, 5)

53 + \
43 ++ \
"\n"

"hello \
there\n"

"""\
Hello \
m'dam\\
"""

"""Start\\
mid\\a\"txt\"
end\\"""

"\n"
