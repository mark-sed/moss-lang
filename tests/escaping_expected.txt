hello
YES
96
hello there
Hello m'dam\
Start\
mid\a"txt"
end\
